lib/ansible/plugins/callback/selective.py

```diff
@@ -41,18 +41,18 @@
 from ansible import constants as C
 from ansible.plugins.callback import CallbackBase
 from ansible.module_utils._text import to_text
-
+from ansible.utils.color import codeCodes
 
 DONT_COLORIZE = False
 COLORS = {
     'normal': '\033[0m',
-    'ok': C.COLOR_OK,
+    'ok': '\033[{}m'.format(codeCodes[C.COLOR_OK]),
```

Comment:
Zero length formats are not supported on python 2.6. Use this instead:

```python
'ok': '\033[{0}m'.format(codeCodes[C.COLOR_OK]),
```

Reply:
Ah, Python 2.6 :)

Looks like that's a general problem throughout this plugin. I can take a look at that.